web/components/application/hacker/pages/PageThree/PageThree.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+
+import { HackerApplicationPageTemplate } from '../PageTemplate';
+import { Page } from '../../../Page';
+import { constraints } from './PageThreeConstraints';
+import { Select } from '../../../../input/select';
+import { Checkbox, CheckboxGroup } from '../../../../input/buttons/CheckboxGroup';
+import { sources } from '../../../sources';
+
+// Emojis
+import robot from '../../../../../assets/emoji/robot.svg';
+import floppy from '../../../../../assets/emoji/floppy.svg';
+import briefcase from '../../../../../assets/emoji/briefcase.svg';
+
+class PageThree extends Page {
+  constructor(props) {
+    super(props);
+    this.constraints = constraints;
+  }
+
+  render() {
+    const {
+      application: {
+        source,
+        isCheckedPrivacyPolicy,
```

Comment:
nit pick but maybe  `isPrivacyPolicyChecked` instead of `isCheckedPrivacyPolicy`? and same for the following three fields?

Reply:
Yep done.